src/Azure/Foundatio.AzureServiceBus.csproj

```diff
@@ -9,8 +9,9 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Foundatio.AzureServiceBus</RootNamespace>
     <AssemblyName>Foundatio.AzureServiceBus</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
```

Comment:
Lets update it to 4.5.2 (https://visualstudiogallery.msdn.microsoft.com/47bded90-80d8-42af-bc35-4736fdd8cd13) as that's the only supported runtime by ms right now and then reinstall all the packages..